README.md

```diff
@@ -25,10 +25,10 @@ Recommended dependencies:
 
 This table lists the tested version of OS/PostgreSQL couples.
 
-| Distribution / PostgreSQL | 9.1                           | 9.4                           | 9.5                | 9.6                | 10                 | 11                 | 12                 |
-| ------------------------- | :---:                         | :---:                         | :---:              | :---:              | :---:              | :---:              | :---:              |
-| Debian 8.x                | :no_entry: :white_check_mark: | :no_entry: :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| Debian 9.x                | :no_entry: :white_check_mark: | :no_entry: :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Distribution / PostgreSQL | 9.4                           | 9.5                            | 9.6                | 10                 | 11                 | 12                 | 13                 |
+| ------------------------- | :---:                         | :---:                          | :---:              | :---:              | :---:              | :---:              | :---:              |
+| Debian 8.x                | :no_entry: :white_check_mark: | :no_entry: :white_check_mark:  | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```

Comment:
Should it be debian 9 and 10 ?

Reply:
:+1: indeed. I've removed the Debian Jessie line.